pipeline/filters/kubernetes.md

```diff
@@ -227,6 +227,31 @@ You can see on [Rublar.com](https://rubular.com/r/HZz3tYAahj6JCd) web site how t
 
 Under certain and not common conditions, a user would want to alter that hard-coded regular expression, for that purpose the option **Regex\_Parser** can be used \(documented on top\).
 
+##### Custom Tag For Enhanced Filtering
+
+One such use case involves splitting logs by namespace, pods, containers or docker id. The tag is restructured within the tail input using match groups, this can simplify the filtring by those match groups later in the pipeline. Since the tag no longer follows the original file name, a custom **Regex\_Parser** that matches the new tag structure is required:
```

Comment:
Can we split sentences on to separate lines to make it easier to review?
We can only add a comment per line so it's clearer then.

Reply:
```suggestion
One such use case involves splitting logs by namespace, pods, containers or docker id.
The tag is restructured within the tail input using match groups, this can simplify the filtering by those match groups later in the pipeline.
Since the tag no longer follows the original file name, a custom **Regex\_Parser** that matches the new tag structure is required:
```

Reply:
Note that `filtering` was misspelt in that big paragraph.